reflex-dom-plaid/Reflex/Dom/Plaid/Link.hs

```diff
@@ -159,28 +169,42 @@ activatePlaidLinkDialog cfg onResult = do
     mkInstitutionFromObj o = do
       name' <- maybeJs valToText =<< o ^. js (t_ "name")
       id' <- maybeJs valToText =<< o ^. js (t_ "institution_id")
-      pure $ case (name', id') of
-        (Just name, Just id_) -> Just PlaidLinkInstitution
-          { _plaidLinkInstitution_name = name
-          , _plaidLinkInstitution_id = id_
-          }
-        _ -> Nothing
+      pure $ PlaidLinkInstitution <$> name' <*> id'
 
     maybeJs f x' = maybeNullOrUndefined x' >>= \case
       Nothing -> pure Nothing
       Just x -> Just <$> f x
 
-    plaidLinkEnvironmentAsText :: PlaidLinkEnvironment -> Text
-    plaidLinkEnvironmentAsText = \case
-      PlaidLinkEnvironment_Sandbox -> "sandbox"
-      PlaidLinkEnvironment_Development -> "development"
-      PlaidLinkEnvironment_Production -> "production"
+    -- plaidLinkEnvironmentAsText :: PlaidLinkEnvironment -> Text
+    -- plaidLinkEnvironmentAsText = \case
+    --   PlaidLinkEnvironment_Sandbox -> "sandbox"
+    --   PlaidLinkEnvironment_Development -> "development"
+    --   PlaidLinkEnvironment_Production -> "production"
```

Comment:
If we don'e need this function, let's remove it.

Reply:
Ok